docs/operating-eck/eck-permissions.asciidoc

```diff
@@ -0,0 +1,150 @@
+:page_id: operator-config
+ifdef::env-github[]
```

Comment:
This page seems identical to the existing `operator-config` page. Also it does not seem to be linked anywhere. Is that maybe a file you forgot to delete?

Reply:
Yes, git mishap, thanks for noticing.